setup-chainctl/README.md

```diff
@@ -9,14 +9,14 @@ and authenticates with it using identity tokens.
 - uses: chainguard-dev/actions/setup-chainctl@main
   with:
     # environment determines the environment from which to download the chainctl
-    # binary from, it is required and has no default (for now).
-    # Required.
-    environment: cookie-monster
+    # binary from.
+    # Optional (default is enforce.dev)
+    environment: enforce.dev
     # audience is the identity token audience to use when creating an identity
     # token to authenticate with Chainguard, it is required and has no default
     # (for now).
-    # Required.
-    audience: oscar-the-grouch
+    # Optional (default is https://issuer.enforce.dev)
+    audience: https://issuer.enforce.dev
```

Comment:
Deprecate this and default to `issuer.{env}` like `dl.{env}`?

Reply:
There might still be value in configuring the audience, even when I want to download chainctl from dl.enforce.dev. I can make it default to issuer.$ENV though.